submissions/adeolaFafemi/Css/signUp.css

```diff
@@ -0,0 +1,498 @@
+    }
+    .navbar .logo img{
+        display: block;
+        height: 50px;
+        margin-top: 30px;
+        margin-left: 20px;
+        margin-bottom: 10px;
+    }
+}
+@media only screen and (max-width:1085px) and (min-width:600px){
+    .navbar{
+        display: flex;
+        align-items:center;
+        background-color:#cc2900;
+        justify-content: space-between;
+    }
+    
+    .navbar li{
+        display: inline-block;
+        list-style-type: none;
+        margin-right: 20px;
+        font-size: 16px;
+        line-height: 19px;
+        color: #ffffff;
+    }
+    .navbar li a{
+        text-decoration: none;
+        color: #ffffff;
+        font-family: 'Montserrat', sans-serif;
+    }
+    .navbar li a:hover{
+        color: #707070;
+    }
+    .navbar .logo img{
+        height: 50px;
+        margin-top: 15px;
+        margin-left: 10px;
+        margin-bottom: 10px;
+    }
+}
+
+/******Sign-up form*******/
+.body-div{
+    display: flex;
+}
+.box0{
+    flex-basis: 50%;
+}
+.reg-notes{
+   margin-top: 190px;
+   margin-left: 70px;
+   margin-right: 100px;
+   font-size: 25px;
+   font-family: 'Montserrat', sans-serif ;
+}
+.reg-notes h1{
+    font-size: 50px;
+}
+.reg-notes h1, .reg-notes a{
+    color:#ffad99 ;
+    font-weight: bold;
+}
+.reg-notes h1, .reg-notes .secondline{
+    line-height: 60px;
+}
+.reg-notes p{
+    color: #5f5f5f;
+}
+.form-type{
+    border: 1px solid #b3b3b3;
+    margin-top: 60px;
+    margin-bottom: 50px;
+    margin-right: 50px;
+    width: 600px;
+    padding-left: 55px;
+    padding-right: 50px;
+    padding-top: 20px;
+    padding-bottom: 30px;
+    border-radius: 23px;
+}
+.form-type label{
+    font-size: 18px;
+    line-height: 22px;
+    color:#5f5f5f;
+    font-family: 'Montserrat', sans-serif ;
+    margin: 10px;
+}
+.form-type .input{
+    width: 500px;
+    height: 40px;
+    border: 2px solid #5f5f5f;
+    border-radius: 10px;
+    margin: 5px;
+}
+.form-type label p{
+    display: inline-block;
+}
+.form-type .checkbox{
+    margin-left: 70px;
+    width: 18px;
+}
+.submit-btn{
+    margin-left: 150px;
+    width: 200px;
+    height: 50px;
+    background-color: #cc2900;
+    border: 1px solid #cc2900;
+    border-radius: 4px;
+    color: white;
+    font-size: 18px;
+    line-height: 12px;
+    font-family: 'Raleway', sans-serif;
+}
+/*******Responsiveness******/
+@media screen and (max-width:600px){
+    .body-div{
+        flex-direction: column;
+        align-items: center;
+    }
+    .reg-notes{
+        margin-top: 20px;
+        margin-left: 40px;
+        margin-right: 40px;
+        font-size: 16px;
+        font-family: 'Montserrat', sans-serif ;
+        text-align: center;
+     }
+     .reg-notes h1{
+        line-height: 30px;
+        font-size: 25px;
+        text-align: center;
+     }
+     .reg-notes h1, .reg-notes .second-line{
+        line-height: 30px;
+    }
+    .form-type{
+        border: 1px solid #b3b3b3;
+        margin-top: 10px;
```

Comment:
Instead of writing `margin-top`, `margin-bottom` and so on. why not just write `margin: 10px 5px 30px 5px` same goes to `padding`. So it is `Top Right Bottom Left`

Reply:
corrected them.
thank you